scripts/deploy

```diff
@@ -1,10 +0,0 @@
-#!/bin/sh
```

Comment:
It was supposed to be replaced by `script/helm/install`, which was added a while ago. I must have forgotten to remove `script/deploy` at the time, or else it may have re-appeared in a merge.

Reply:
They are different though as `helm/install` provides sql and a message broker rather than just the application.

Reply:
True, I think those should be put behind a `--link` flag in line with the `start` script, rather than keeping `deploy` as it seems very unclear what each script is for at the moment.

Reply:
Thinking more about this (and remembering back to when I added the `helm/` scripts), the reason the scripts are different is that the `deploy` script wasn't working.

The app requires a database and message queue to function but we don't have anything in the dev repo to deploy a shared message queue or database for the services to connect to. Therefore, each app needs to deploy its own for local deployment testing.

I think for that reason it's sensible to remove the outdated `deploy` script and not do any more work on the `install` script in this PR.